blocks/init/src/Blocks/components/embed/components/embed-editor.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classnames from 'classnames';
+import { video } from '@wordpress/icons';
+import { Placeholder } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { selector, checkAttr } from '@eightshift/frontend-libs/scripts';
+import manifest from '../manifest.json';
+
+export const EmbedEditor = (attributes) => {
+	const {
+		componentClass,
+	} = manifest;
+
+	const {
+		selectorClass = componentClass,
+		blockClass,
+		additionalClass,
+	} = attributes;
+
+	const embedUse = checkAttr('embedUse', attributes, manifest);
+	let embedUrl = checkAttr('embedUrl', attributes, manifest);
+
+	const embedClass = classnames([
+		selector(componentClass, componentClass),
+		selector(blockClass, blockClass, selectorClass),
+		selector(additionalClass, additionalClass),
+	]);
```

Comment:
```suggestion
	const embedClass = classnames(
		componentClass,
		bem(blockClass, selectorClass),
		additionalClass
	);
```
Import `classnames` and `bem` from Frontend libs